### DISCUSSION TITLE
blog/api-design-in-bff

### DISCUSSION THREAD

**Discussion by giscus** [notionpresso/nextjs-blog-template#5] [Announcements]
# blog/api-design-in-bff

Exploring the shift from domain-centric to data-centric API endpoints after introducing BFF. This article introduces BFF and DDD concepts, then proposes API endpoint design strategies in the context of the BFF pattern.

http://localhost:3000/blog/api-design-in-bff

**Comment by leedc0101** [comment]
asdfsdaf

**Comment by anonymousRecords** [comment]
@leedc0101 
Since the giscus settings in the template are linked to this repository, any test comments will appear here.
Please change `repo="notionpresso/nextjs-blog-template"` to your repository ([reference](https://github.com/notionpresso/nextjs-blog-template/blob/8a7dcd132a4425b04d4848fd47d569c5201035c6/components/comment.tsx#L8))
